README.md

```diff
@@ -155,6 +155,21 @@ To set up the mediator storage (MongoDB):
 - `MONGODB_PASSWORD` - is the password used by the Mediator service to connect to the database.
 - `MONGODB_DB_NAME` - is the name of the database used by the Mediator.
 
+#### Mediator storage 
+- The `messages` collection contains two types of messages: `Mediator` and `User`.
+1. **Mediator Messages**:
+    - These messages received by mediator for any interactions with the mediator.
+    - Examples include messages for setting up mediation, requesting mediation, or picking up messages from the mediator.
+    - These messages stored in collection can be used for debugging purpose mediator functionality and interactions with the mediator. Hence they can be deleted after a period of time.
+    - This message type `Mediator` can be setup to have a configurable Time-To-Live (TTL) value, after which they can expire.
+    - This is how the TTL can be configured for the collection messages [initdb.js](initdb.js)
+2. **User Messages**:
+    - These are the actual messages e.g like the Forward message from the mediator, contain a `User` message inside.  This inside message is stored as type `User` to be delivered to user.
```

Comment:
- These are the actual messages, e.g. the Forward message from the mediator, containing a `User` message inside. This inside message gets stored as `User` deliverable to the user.